Source/Core/Core/Config/MainSettings.cpp

```diff
@@ -143,4 +143,9 @@ const ConfigInfo<std::string> MAIN_RESOURCEPACK_PATH{{System::Main, "General", "
                                                      ""};
 const ConfigInfo<std::string> MAIN_FS_PATH{{System::Main, "General", "NANDRootPath"}, ""};
 const ConfigInfo<std::string> MAIN_SD_PATH{{System::Main, "General", "WiiSDCardPath"}, ""};
+
+// Main.Network
+
+const ConfigInfo<int> MAIN_NETWORK_TIMEOUT{{System::Main, "Network", "NetworkTimeout"},
+                                           190};  // Based on https://dolp.in/pr8759 hwtest
```

Comment:
You may wish to relocate this comment on top of the definition so that the wonky formatting can be avoided.

The comment can just be formatted to indicate it means the default value.

```cpp
// Default value based on https://dolp.in/pr8759 hwtest
```